website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -365,3 +365,7 @@ models:
 ```
 </File>
 
+## Model contracts
+
+The `dbt-trino` adapter supports [Model contracts](/docs/collaborate/govern/model-contracts). However, currently only `not_null` [constraint](/reference/resource-properties/constraints) is supported.
+Please ensure that underlying connector supports `not null`, before using this constraint in your model.
```

Comment:
```suggestion
Before using a constraint in your model, make sure the underlying connector supports `not null`.
```

Reply:
curious, what happens if it doesn't? what sort of issues will the user encounter if they do end up using it? it's good practice to inform people _why_ they should do a thing as it helps with learning.

Reply:
While trying `not_null` constraint on connector which doesn't support it, user will receive Trino error at dbt run stage, e.g.:
`Hive tables do not support NOT NULL columns`
Should I add detailed description in docs what would happen then? I think that it is not necessary in this case, as making sure that you use supported feature is self explanatory IMHO - you can't use feature, which is not supported 🙂 
WDYT?